### ISSUE TITLE
Magnifying-Glass 100% while Left Mouse Button Held

### ISSUE THREAD

**Issue by djlongy** [d2phap/ImageGlass#1425]
### Is your feature request related to a problem? Please describe.
When viewing an image that is larger than the resolution of the monitor, it is action intensive to view at move around the image at 100% scaling.
A quicker way to view images at 100% while going through images exists and can be implemented to ImageGlass.

### Describe the solution you'd like
Currently the way FastPictureViewer handles this is quite good and is very intuitive.

Images that are larger than the current resoultion can be automatically displayed as Auto-Fit to the screen boundaries.
Holding the left mouse button in an area will automatically zoom the image to 100% in the center of that area while the button is held down cropping the rest of the image that is out of the window/screen bounds.
Moving the mouse while keeping the button held down will reveal/show  the other parts of the image as you move the mouse around to those areas. This would effectively act as a magnifying glass at 100% (customisable) while the left mouse button is held down.

### Describe alternatives you've considered
nil, FastPictureViewer does this excellently.

### Additional context
This allows users to not have to click any buttons or press any keys to change the zoom, and it allows for quickly going through multiple images in a folder.
They can effectively navigate different images and view 100% scaling with only the use of the mouse scroll (next/previous) and left mouse button to magnify at 100% while the left mouse button is held.

**Comment by KoalaBear84** [comment]
Yes, that is a great feature I use from Faststone Image Viewer. The only reason I'm looking for an alternative is because I have folders with 15.000 images, and Faststone takes 3-4 seconds after start because it becomes responsive to input because of the amount of images, which it shouldn't. :)
